platform-tests/src/test/java/org/junit/platform/launcher/TestIdentifierTests.java

```diff
@@ -63,15 +63,6 @@ void currentVersionCanBeSerializedAndDeserialized() throws Exception {
 		assertDeepEquals(originalIdentifier, deserializedIdentifier);
 	}
```

Comment:
I think we should parameterize this test with the number of tags so serialization is tested with 0, 1, and 2 tags to ensure `copyOf()` actually always returns `Serializable` implementations of `Set`. I can do it as a follow-up, though.